docs/changelog.rst

```diff
@@ -2,6 +2,9 @@
 Changelog
 =========
 
+* :release:`7.9.6 <2021-10-18>`
+* :feature:`316` Add new container: `96-pcr-fs-clear`
```

Comment:
@joshhacksthings update description of feature you're planning release with the change